core/app/models/spree/product.rb

```diff
@@ -326,6 +326,8 @@ def category
     end
 
     def taxons_for_store(store)
+      return if store.blank?
```

Comment:
@sebastian-palma why do we need this change?

Reply:
That's because when using the product serializer from the [HasWebhooks](https://github.com/spree/spree/blob/5d189d7c068954632576d89bde92adf5a6f382fe/api/app/models/concerns/spree/webhooks/has_webhooks.rb#L39) concern, the serializer uses the product `taxons_for_store` expecting to receive a store from the params but we don't have a store at that point. In main, the code for the serializer options is set within the controller, but the code for the webhooks doesn't involve a controller.

I'm still unaware of what store should be there @damianlegawiec, might be a default one?

Reply:
@sebastian-palma ah, gotcha - in that case, let's pull taxons from all stores